core/trino-spi/src/main/java/io/trino/spi/ptf/EmptyTableFunctionHandle.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.spi.ptf;
+
+public enum EmptyTableFunctionHandle
```

Comment:
The intention behind this class was to provide a default empty implementation that functions can use if they find it convenient, instead of defining their own "empty handle" class. It simplifies the `analyze()` method, as you can skip passing the handle.

Reply:
Not sure I understand "skip passing the handle". The handle represents the identity of the function, so what does it mean to skip it?